README.md

```diff
@@ -34,11 +34,17 @@ Use it in your lambda:
 ```javascript
 const { pushStream } = require('dynamodb-stream-elasticsearch');
 
-const { ES_ENDPOINT, INDEX, TYPE } = process.env;
+const { ES_ENDPOINT, INDEX, TYPE, REFRESH } = process.env;
 
 function myHandler(event, context, callback) {
   console.log('Received event:', JSON.stringify(event, null, 2));
-  pushStream({ event, index: INDEX, type: TYPE, endpoint: ES_ENDPOINT })
+  
+  pushStream({ 
+      event, 
+      index: INDEX, // Optional - fallbacks to the table name of your dynamo instance
+      type: TYPE, // Optional - fallbacks to index and if that dosn’t exist the table name of your dynamo instance
+      refresh: REFRESH, // Optional - Defaults to true
```

Comment:
I guess you don't need to add this description, it is already in the table below.